scrapy_autounit/utils.py

```diff
@@ -215,12 +216,15 @@ def write_test(path, test_name, fixture_name, encoding, url):
 
 class AutoUnit(unittest.TestCase):
     def test__{test_name}__{fixture_name}(self):
+        files = os.listdir(os.path.abspath(Path(__file__).parent))
+        files = [f for f in files if f.endswith('.bin')]
         self.maxDiff = None
-        file_path = (
-            Path(__file__).parent / '{fixture_name}.bin'
-        )
-        test = generate_test(file_path.resolve(), '{encoding}')
-        test(self)
+        for f in files:
```

Comment:
Maybe we could add a message indicating which fixture is being tested at the moment so in case it fails, the user would know where to look at.
We don't have a tool to extract data from fixtures yet but we're already working on it.